src/bin/rustfmt.rs

```diff
@@ -122,6 +122,12 @@ fn make_opts() -> Options {
 
     opts.optflag(
         "",
+        "unstable-features",
+        "Enables unstable features. Only available in rust nightlies",
```

Comment:
nit: could you change the text here to end with `Only available on the nightly channel` please